README.md

```diff
@@ -56,6 +56,9 @@ To prepare your system to deploy Clearwater using Compose, after running the com
     # Build the base Clearwater docker image.
     cd clearwater-docker
     sudo docker build -t clearwater/base base
+    sudo docker-compose -f minimal-distributed.yaml build
```

Comment:
Do you need to add a similar line in the 'Cleaning Up' section first paragraph related to 'To rebuild an image....'?